tests/e2e/routing.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { test, expect } from "@playwright/test";
+
+const BASE_URL = "/dengue/may24";
+const expectIndexPage = async (
+    page,
+    url: string,
+    selectedIndicator: string,
+    selectedCountry: string,
+    scale: string,
+    featureCount: number,
+    selectedCountryFeatureCount: number
+) => {
+    await page.waitForURL(new RegExp(`${BASE_URL}${url}`));
+    await expect(await page.textContent("button.bg-blue")).toBe(selectedIndicator);
+    const summary = await page.locator(".choropleth-data-summary");
+    await expect(await summary).toHaveAttribute("selected-indicator", selectedIndicator);
```

Comment:
Using more forgiving form of this test, which doesn't require attribute value to be present immediately.

Reply:
...ugh, and had further e2e test misery, where one test started consistently failing on Safari only, without any substantive code changes.. Need to test on a real Safari instance really, hopefully when Rich is back tomorrow can try with his sacrifical test Mac. Have temporarily removed the Safari e2e config, and made ticket to investigate and reinstate: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5274/Investigate-Safari-e2e-tests-failures